bot/exts/utils/utils.py

```diff
@@ -86,36 +86,38 @@ def get_info(char: str) -> Tuple[str, str]:
         await LinePaginator.paginate(char_list, ctx, embed, max_lines=10, max_size=2000, empty=False)
 
     @command()
-    async def zen(self, ctx: Context, *, search_value: Union[int, str, None] = None) -> None:
+    async def zen(self, ctx: Context, zen_rule_index: int | None,
+                  *, search_value: Union[int, str, None] = None) -> None:
```

Comment:
Better to use the style mentioned in our style guide https://www.pythondiscord.com/pages/guides/pydis-guides/contributing/style-guide/#bracket-and-item-arrangement
```suggestion
async def zen(
    self, 
    ctx: Context,
    zen_rule_index: int | None,         
    *, 
    search_value: Union[int, str, None] = None
) -> None:
```

Reply:
You are definitely right, my bad.
Applied the suggestion